status/templatetags/status_filters.py

```diff
@@ -121,10 +132,7 @@ def show_detail(submission, user):
     contests = get_running_contests()
```

Comment:
I also want to discuss another respect.
Is it proper to allow contest owner / coowner view contestants' detail in the other contest duration?